static/_redirects

```diff
@@ -113,6 +113,7 @@
 /addons/                                    /integrations/                                                                                      301
 /recipes/                                   /integrations/                                                                                      301
 /addons/tag/*                               /integrations/tag/:splat                                                                            301
+/docs/react/addons/addon-catalog/           /docs/react/integrations/integration-catalog/                                                       301
```

Comment:
Whoops. I totally gave you the wrong guidance in #439 🤦‍♂️. `/docs` redirects have some special considerations, due to the dynamic framework and version (or lackthereof). I'll push an update tomorrow, as that's much easier than attempting to explain.